packages/elcollectooorr/contracts/token_vault/tests/test_contract.py

```diff
@@ -38,8 +38,6 @@
     TokenVaultFactoryContract,
 )
 
-from tests.conftest import ROOT_DIR
-
 
 DEFAULT_GAS = 1000000000
```

Comment:
There is a problem with this test file, it depends on `basket_factory` and `basket`. These integration tests need to move up to the agent level, as we don't want to change the dependencies unnecessarily just for the tests.